translations/Indonesian/general-rules.md

```diff
@@ -0,0 +1,26 @@
+# General Rules
+
+
+**Should translations be formal or informal?**
+Translations should be formal.
+
+**Which grammatical person should be used for translations (e.g. second person singular, first person plural)?**
+Use grammatical person as appropriate to the language. 

```

Comment:
Is there any preference here? E.g. is one form more polite / appropriate than another?